pkg/status/status.go

```diff
@@ -85,10 +87,45 @@ func GetStatus() (map[string]interface{}, error) {
 	}
 
 	stats["systemProbeStats"] = getSystemProbeStats()
+	stats["apmStats"] = GetAPMStatus()
 
 	return stats, nil
 }
 
+// GetAPMStatus returns a set of key/value pairs summarizing the status of the trace-agent.
+// If the status can not be obtained for any reason, the returned map will contain an "error"
+// key with an explanation.
+func GetAPMStatus() map[string]interface{} {
+	port := 8126
+	// TODO(gbbr): This should be handled by the shared config package once
+	// we migrate APM env. vars there.
+	if p, ok := os.LookupEnv("DD_APM_RECEIVER_PORT"); ok {
+		if v, err := strconv.Atoi(p); err == nil {
+			port = v
+		}
+	}
+	if config.Datadog.IsSet("apm_config.receiver_port") {
+		port = config.Datadog.GetInt("apm_config.receiver_port")
+	}
+	url := fmt.Sprintf("http://localhost:%d/debug/vars", port)
+	resp, err := (&http.Client{Timeout: 1 * time.Second}).Get(url)
```

Comment:
I think the constant should be move to the config so customer can change it if needed (would be helpful when troubleshooting with support). We don't need to document it in the datadog.yaml I think.

Reply:
Since this only applies to local environments, I still don't think this warrants a config setting. Also, `1s` is too short, we need at least `2s`. We know from prior experience that even locally `1s` is sometimes not enough.